tests/fc/layer_example.c

```diff
@@ -131,7 +155,7 @@ int main(int argc, char* argv[])
   if (argc > i) bn         = atoi(argv[i++]);
   if (argc > i) bk         = atoi(argv[i++]);
   if (argc > i) bc         = atoi(argv[i++]);
-  if (argc > i) prec_bf16  = atoi(argv[i++]);
+  if (argc > i) prec       = atoi(argv[i++]);
```

Comment:
better to use char and then mapping to LIBXSMM_DATETYPE as this solution won't allow for HF8